### PR TITLE
Improve the list items on narrow screens

### PR DESCRIPTION
**Pull request by HFaaaaa**
#101 
Improve the items of the list on narrow screens in order to display more cells at once.

### CODE REVIEW

File: app/src/app/list/polls-table/polls-table.component.sass [hunk 1]
```diff
@@ -62,24 +66,25 @@ table
...
         padding: $tiny-space
         display: block
         text-align: right
         text-overflow: ellipsis
         overflow: hidden
         white-space: nowrap
+      td.title
+        border-bottom: solid 2px #ddd
+        text-align: left
+        font-weight: bolder
```
Comment (Jogo27) [inline app/src/app/list/polls-table/polls-table.component.sass:78]: Looks too heavy to me. I prefer regular weight.

File: app/src/app/list/polls-table/polls-table.component.sass [hunk 2]
```diff
@@ -62,24 +66,25 @@ table
...
         &:before
           content: attr(data-label)
           float: left
           font-weight: bold
           padding-right: 1.5em
-      td.round,  td.deadline,  td.terminated
-        text-align: right
-      td.action
+      td.action-manage
+        display: block
```
Comment (Jogo27) [inline app/src/app/list/polls-table/polls-table.component.sass:88]: Seems wrong to me. This results in wrong aligment.
image
I suggest:
```suggestion
        justify-content: end
```

**Review by Jogo27 — CHANGES_REQUESTED** [review]

**Comment by HFaaaaa** [comment]
PTAL

File: app/src/app/list/polls-table/polls-table.component.sass [hunk 3]
```diff
@@ -62,24 +66,23 @@ table
...
+      td.title
+        border-bottom: solid 2px #ddd
+        text-align: left
+      td.round,  td.deadline,  td.terminated
+        font-size: smaller
+        text-align: right
         &:before
           content: attr(data-label)
           float: left
           font-weight: bold
```
Comment (Jogo27) [inline app/src/app/list/polls-table/polls-table.component.sass:84]: Sorry to not have noticed that before, but `bold` is an almost deprecated font-weight. You should use numeric values from 100 to 900. The font used here (Asap) is imported from Google Fonts with weights 400, 500 and 600 only. If you want a bolder weight, you must add it at line 10 of app/src/index.html.

**Review by Jogo27 — CHANGES_REQUESTED** [review]
It's very nice, congratulations. I'm very sorry to not have seen the bold issue below. Could you also please add an empty line before line 4 of  app/src/app/list/polls-table/polls-table.component.sass?

**Comment by HFaaaaa** [comment]
PTAL

**Review by Jogo27 — APPROVED** [review]
LGTM

**Pull request merged**